djracula/templates/admin/base_site.html

```diff
@@ -3,7 +3,7 @@
 {% block title %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}
 
 {% block branding %}
-<h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header|default:_('Django administration') }}</a></h1>
+<h1 id="site-name"><a href="{% url 'admin:index' %}">Djracula</a></h1>
```

Comment:
@taranjeet  I think we have to change the site.py to change these headers value dynamically. Suggestions?